INSTALL

```diff
@@ -151,20 +169,60 @@ BlueGene/Q and point `-Dwith-python=<...>` to a valid python version for cross
 compilation:
 
     -Dwith-python=/bgsys/tools/Python-2.7/bin/hostpython
+or (much better)
+    -Dwith-python=/bgsys/local/python3/3.4.2/bin/python3
 
-Eventually, you also have to specify `PYTHON_LIBRARY` and `PYTHON_INCLUDE_DIR`,
-if the they are not found, e.g.:
+CMake <3.4 is buggy about finding the matching libraries (for many years).
+Thus, you also have to specify `PYTHON_LIBRARY` and `PYTHON_INCLUDE_DIR`
+if they are not found OR the incorrect libraries are found, e.g.:
 
     -DPYTHON_LIBRARY=/bgsys/tools/Python-2.7/lib64/libpython2.7.so.1.0
     -DPYTHON_INCLUDE_DIR=/bgsys/tools/Python-2.7/include/python2.7
+or (much better)
+    -DPYTHON_LIBRARY=/bgsys/local/python3/3.4.2/lib/libpython3.4m.a
+    -DPYTHON_INCLUDE_DIR=/bgsys/local/python3/3.4.2/include/python3.4m
+
+A complete `cmake` line for PyNEST could look like this:
+
```

Comment:
I assumed so as single backticks were already used in other places of the file. I'd say that the triple ones at least don't hurt readability in a non-markdown enabled environment and help if rendered correctly.